pyxrf/model/fileio.py

```diff
@@ -1749,7 +1749,7 @@ def _make_hdf(fpath, runid, full_data=True,
         if full_data == True:
             sc.export(hdr, fpath, db.mds, fields=fds, use_uid=False)
 
-    elif hdr_tmp.start.beamline_id == 'xf05id':
+    elif hdr_tmp.start.beamline_id == 'xf05id' or str(hdr_tmp.start.beamline_id) == 'SRX':
```

Comment:
It would probably help with adoption if this were configurable by some means other than a PR.